examples/nextjs-with-typescript/pages/SimpleVideoPlayer.tsx

```diff
@@ -0,0 +1,64 @@
+import Head from 'next/head';
```

Comment:
This is a test page for using the `<SimpleVideoPlayer/>` component. Works similar to our `<MuxPlayer/>` test page though not designed for similar robustness. Expect to have to apply parameters and then load the resultant URL for expected behavior.